Lib/Protocols/IdRegister.pas

```diff
@@ -323,7 +316,6 @@ implementation
   IdUserAccounts,
   IdUserPassProvider,
   IdVCard,
-  IdWebDAV,

```

Comment:
Removing `IdWebDAV` causes an error in Delphi, as `TIdWebDAV` is being registered in Delphi but not in FPC. So either `IFDEF` this unit or fix the registration of `TIdWebDAV` for FPC.